x-pack/plugins/endpoint/public/embeddables/resolver/view/index.tsx

```diff
@@ -24,6 +25,8 @@ export const AppRoot = React.memo(({ store }: { store: Store<ResolverState, Reso
   );
 });
 
+const bgColor = '#222222';
```

Comment:
most of the other colors are defined in defs.tsx. Maybe move this there? or group it w/ the other color constants in some file?

Reply:
Good point - Changed to an import from defs.